src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let robotAccountantsCreated = 0;
```

Comment:
```suggestion
  let callsAmount = 0;
```
I think it's better to find a more proper name for counter